package.json

```diff
@@ -24,7 +24,13 @@
   },
   "homepage": "https://github.com/Cryptonomic/ConseilJS#readme",
   "dependencies": {
+    "@ledgerhq/hw-transport": "4.21.0",
+    "@ledgerhq/hw-transport-node-hid": "^4.22.0",
     "awesome-typescript-loader": "^5.0.0",
+    "aws-sdk": "^2.326.0",
+    "babel-runtime": "6.26.0",
+    "basil-tezos-ledger": "0.0.1",
```

Comment:
Let's not forget to replace this with an official unofficial Cryptonomic Ledger.js NPM package. :grin: 

We can set up the new package together in person.